src/FlubuCore.Analyzers.Tests/FlubuCore.Analyzers.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Turned out the test project has to target .NET Core 2.1 or 3.x.
Ref: https://stackoverflow.com/questions/54770830/unable-to-find-testhost-dll-please-publish-your-test-project-and-retry

I noticed that there are lots of differences in bin/[config]/FlubuCore.Analyzers.Tests.deps.json between netcoreapp2.0 and netcoreapp2.1. Not sure whether it's relevant or not.